src/answers-headless.ts

```diff
@@ -48,6 +48,10 @@ export default class AnswersHeadless {
     this.stateManager.dispatchEvent('query/setSource', source);
   }
 
+  setSearchAggregationId(uuid: string): void {
+    this.stateManager.dispatchEvent('query/setSearchAggregationId', uuid);
```

Comment:
if somebody wanted to clear the id would they just set it to blank string?

Reply:
yeah, do you think undefined/null is better?

Reply:
eh think it's fine!